apps/ior/ior.slurm

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
This script is the same than the ior.sh, can the #SBATCH lines be added on the SLURM command line so we keep a single script and not one per job scheduler ?

Reply:
This is actually one thing I wanted to ask. For simplicity of testing I went ahead and created separate scripts for Slurm, but that generates one additional file that needs to be maintained. I can work on adding the necessary logic in the current bash scripts to correctly manage both PBS and Slurm.